lib/main.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'Home.dart';
+import 'Login.dart';
+import 'Upcoming.dart';
+import 'CurrentRide.dart';
+import 'Schedule.dart';
+import 'BottomMenu.dart';
+import 'Map.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Carriage',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: MyHomePage(title: 'Carriage Home Page'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _MyHomePageState createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  int _counter = 0;
```

Comment:
Remove all of the code that is from the tutorial that doesn't pertain to this project (for example, the incrementing/decrementing counter). If you need some sort of content to make Flutter run, add text saying "more content coming soon".